map_maker/__init__.py

```diff
@@ -1,11 +1,11 @@
 """
-Copyright 2020 Expedia Group
+Copyright 2020 Expedia, Inc
```

Comment:
```suggestion
Copyright 2020 Expedia, Inc.
```
^ they need a closing fullstop, I see this is missing at https://github.com/ExpediaGroup/new-project/#apache-header so I'll fix it there shortly.

Obviously applies to the rest below too.

Reply:
Ah okay. I'll fix that.